{"status": "failed", "name": "Fail when sorting on variable removed by DISTINCT", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "skip"}, {"name": "feature", "value": "ReturnAcceptance2"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython2"}], "stop": 1578486130431, "historyId": "d5c5aae76c9db2b1014ca6ce1a9b2063", "start": 1578486130215, "steps": [{"status": "passed", "start": 1578486130216, "stop": 1578486130422, "name": "Given an empty graph"}, {"status": "passed", "start": 1578486130422, "stop": 1578486130426, "name": "And having executed", "attachments": [{"source": "96ab1e83-e423-4d18-ab08-3be7314da4fb-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "passed", "start": 1578486130426, "stop": 1578486130429, "name": "When executing query", "attachments": [{"source": "45adddc7-7564-4597-9afa-0b555d319a7e-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "failed", "start": 1578486130429, "stop": 1578486130430, "name": "Then a SyntaxError should be raised at compile time: UndefinedVariable", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 116, in step_impl\n    assert exception != None\n"}}], "statusDetails": {"message": "AssertionError\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 116, in step_impl\n    assert exception != None\n"}, "uuid": "bbf81cbf-9f25-475e-a8a9-3708683c8cdf"}